beacon-chain/p2p/service.go

```diff
@@ -75,3 +78,19 @@ func (s *Service) Status() error {
 	}
 	return nil
 }
+
+// Encoding returns the configured networking encoding.
+func (s *Service) Encoding() encoder.NetworkEncoding {
+	return nil
+}
+
+// PubSub returns the p2p pubsub framework.
+func (s *Service) PubSub() *pubsub.PubSub {
```

Comment:
`Broadcast` has its own `broadcaster.go`, do you want to put this in `pubsuber.go` as well?

Reply:
These are getter methods. I put Broadcast into its own file because it will be non-trivial logic with its own tests.